sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -564,4 +569,9 @@
    */
   String IMPORT_UNKNOWN_FILES_KEY = "sonar.import_unknown_files";
 
+  /**
+   * @since 5.1
+   */
+  String DEFAULT_ISSUE_ASSIGNEE = "sonar.assign.default";
```

Comment:
the last part "default" is misleading. Is it a boolean or a string ?
I would rename sonar.assign.defaultLogin (is it a login ?)